nuxt.config.js

```diff
@@ -204,6 +208,7 @@ export default {
     dsn:
       process.env.SENTRY_DSN ||
       'https://3f3e05dbe6994c318d1bf1c8bfcf71a1@o288582.ingest.sentry.io/1525413',
+    disabled: process.env.NODE_ENV !== 'production',
```

Comment:
This seems to be incorrectly committed from the Sentry PR.

Reply:
Yes! Thank you for catching this, @dhruvkb !